index.html

```diff
@@ -62,16 +62,36 @@ <h5 class="card-title">Connect your own Workspace</h5>
       <div class="row">
         <div class="col">
           <details class="mb-2">
-            <summary class="mb-3">What is the remote academia shared channel?</summary>
-            <p>Coming soon</p>
+              <summary class="mb-3"><b>What is the Remote Academia community?</b></summary>
+            <p>
+            Remote Academia started in response to changes triggered by the COVID-19 pandemic. As many educational institutions
```

Comment:
How we started isn't the same as what we are. 
If people read one sentence, they should know at the end.